scipy/linalg/meson.build

```diff
@@ -66,9 +66,12 @@ fblas_module = custom_target('fblas_module',
 # LAPACK - we have historically put these in `_fblas`.
 py3.extension_module('_fblas',
   fblas_module,
+  fortran_args: _fflag_lp64,
   link_args: version_link_args,
   dependencies: [lapack_dep, blas_dep, fortranobject_dep],
+  link_with: [g77_abi_wrappers],
   install: true,
+  link_language: 'fortran',
```

Comment:
Question: why did this change? It seems wrong, this should be a pure C module.

Reply:
No reason apparently. Is likely a leftover from past iterations. Removed now, thanks!